docs/content.zh/docs/dev/python/datastream/operators/overview.md

```diff
@@ -163,6 +163,29 @@ ds.map(lambda i: (i[0] + 1, i[1]), Types.TUPLE([Types.INT(), Types.STRING()])) \
 
 Generally, the output type needs to be specified for the map operation in the above example if the sink only accepts special kinds of data, e.g. Row, etc.
 
+## Operator Chaining
+
+By default, multiple non-shuffle Python functions will be chained to avoiding the serialization and
+deserialization and improving the performance. There are also cases where you may want to disable
```

Comment:
```suggestion
By default, multiple non-shuffle Python functions will be chained together to avoid the serialization and
deserialization to improve the performance. There are also cases where you may want to disable
```